app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def col_width
+    width =
```

Comment:
There is no need to assign a variable as it is the value returned. Ruby returns the last thing run in a method. 
```
def col_width
  params[:controller] == 'admin/welcome' ? 15 : 16.25
end
```